packages/victory-core/src/victory-util/axis.js

```diff
@@ -209,13 +186,56 @@ function getTickArray(props) {
     ticks = stringMap ? tickValues.map((tick) => stringMap[tick]) : range(1, tickValues.length + 1);
   }
   const tickArray = ticks ? uniq(ticks) : getTicksFromFormat(props);
-  const filterArray = (arr) => {
+  const buildTickArray = (arr) => {
+    const newTickArray = [];
     const domain = (props.domain && props.domain[axis]) || props.domain;
-    return Array.isArray(domain)
-      ? arr.filter((t) => t >= Math.min(...domain) && t <= Math.max(...domain))
-      : arr;
+    if (arr) {
+      arr.forEach(function (t, index) {
+        if (Array.isArray(domain)) {
+          if (t >= Math.min(...domain) && t <= Math.max(...domain)) {
```

Comment:
I think this should use Collection.getMinValue rather than Math.min so that dates are handled correctly.